### PR TITLE
Adding action project update integration tests

### PR DESCRIPTION
**Pull request by lancewf**
### :nut_and_bolt: Description: What code changed, and why?

Adding API level integration tests for client runs action project re-tagging.

### :chains: Related Resources
#2192 

### :+1: Definition of Done

Have a test for each possible attribute for event project update re-tagging.

### :athletic_shoe: How to Build and Test the Change

Check the buildkite tests

### :white_check_mark: Checklist

- [x] Tests added/updated?
- [ ] Docs added/updated?

### :camera: Screenshots, if applicable

### CODE REVIEW

File: e2e/cypress/integration/api/iam/10_projects_api_action_update.ts [hunk 1]
```diff
@@ -0,0 +1,157 @@
+import { describeIfIAMV2p1 } from '../../constants';
```
Comment (bcmdarroch) [inline e2e/cypress/integration/api/iam/10_projects_api_action_update.ts]: just noticed, these should be `.spec.ts` files, not `.ts`

Comment (lancewf) [reply e2e/cypress/integration/api/iam/10_projects_api_action_update.ts]: Thank you! I could not figure out why they looked different in VS code. 
Screen Shot 2019-11-20 at 4 44 28 PM

Comment (lancewf) [reply e2e/cypress/integration/api/iam/10_projects_api_action_update.ts]: I renamed them all in the three PRs I have out.

File: e2e/cypress/integration/api/iam/10_projects_api_action_update.spec.ts [hunk 2]
```diff
@@ -0,0 +1,157 @@
+import { describeIfIAMV2p1 } from '../../constants';
+import { uuidv4 } from '../../helpers';
+
+describeIfIAMV2p1('Action project update tagging', () => {
+  const cypressPrefix = 'test-client-runs-update';
+
+
```
Comment (blakestier) [inline e2e/cypress/integration/api/iam/10_projects_api_action_update.spec.ts]: extra line 🤷‍♂

File: e2e/cypress/integration/api/iam/10_projects_api_action_update.spec.ts [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+    cy.cleanupV2IAMObjectsByIDPrefixes(cypressPrefix, ['projects', 'policies']);
+  });
+
+  after(() => cy.cleanupV2IAMObjectsByIDPrefixes(cypressPrefix, ['projects', 'policies']));
+
+  it('tagging 2 projects with all the attributes on a client runs action', () => {
+    const actionId = uuidv4();
+    const entityName = `entity_name-${Cypress.moment().format('MMDDYYhhmmss')}`;
+
+    // Ingest an action with attribues that match all the projects
```
Comment (blakestier) [inline e2e/cypress/integration/api/iam/10_projects_api_action_update.spec.ts]:
```suggestion
    // Ingest an action with attributes that match all the projects
```

Comment (lancewf) [reply e2e/cypress/integration/api/iam/10_projects_api_action_update.spec.ts]: Not again :)

**Pull request merged**